.github/workflows/deploy.yml

```diff
@@ -1,19 +1,9 @@
 name: Deploy Prod
 
 on:
-  # TODO: deploy automatically when we are confident in this.
-  # push:
-  #   tags: ["v*"]
+  push:
+    branches: ["main"]
   workflow_dispatch:
-    inputs:
```

Comment:
we can't actually use these inputs because of how we've configured the ref trusted for GHA-based deployment.